### PR TITLE
edgex: upgrade edgex lib version and go version

### PR DESCRIPTION
**Pull request by jinfahua**

### CODE REVIEW

**Comment by jinfahua** [comment]
@ngjaying @EMQmyd Please upgrade your local GO version to 1.15...

File: .ci/Dockerfile-plugins-alpine [hunk 1]
```diff
@@ -16,4 +16,4 @@ RUN set -e -u -x \
     && go build --buildmode=plugin -o _plugins/$PLUGIN_TYPE/$PLUGIN_NAME/$PLUGIN_NAME@$VERSION.so plugins/$PLUGIN_TYPE/$PLUGIN_NAME.go \
     && if [ -f "etc/$PLUGIN_TYPE/$PLUGIN_NAME.yaml" ]; then cp etc/$PLUGIN_TYPE/$PLUGIN_NAME.yaml _plugins/$PLUGIN_TYPE/$PLUGIN_NAME; fi \
     && cd _plugins/$PLUGIN_TYPE/$PLUGIN_NAME \
-    && zip -r ${PLUGIN_NAME}_$(go version -v | grep -o "linux/.*" | sed -r 's linux/(.*) \1 g').zip .
+    && zip -r ${PLUGIN_NAME}_$(go version | grep -o "linux/.*" | sed -r 's linux/(.*) \1 g').zip .
```
Comment (ngjaying) [inline .ci/Dockerfile-plugins-alpine:19]: Is "-v" removed intentionally?

Comment (jinfahua) [reply .ci/Dockerfile-plugins-alpine:19]: Yes, with Go 15.1, `go version -v` throws an error. @zhanghongtong

**Comment by zhanghongtong** [comment]
@jinfahua 
The specific version of goang:1.5 is 1.15.2 instead of 1.15.1, and the version number of golang:1.5 will be updated with the update of golang, is this okay?

**Comment by jinfahua** [comment]
> @jinfahua
> The specific version of `goang:1.5` is `1.15.2` instead of `1.15.1`, and the version number of `golang:1.5` will be updated with the update of golang, is this okay?

I suppose `goang:1.5` should be ``goang:1.15`? So I would suggest to use the specific version # `1.15.1`.

**Review by zhanghongtong — APPROVED** [review]

**Pull request merged**